modules/service/src/main/resources/db/migration/V0839__time_discount.sql

```diff
@@ -0,0 +1,50 @@
+-- Discriminator for various time accounting charge discount types.
+CREATE TYPE e_time_charge_discount_type AS ENUM (
+  'daylight',
+  'fault',
+  'nodata',
+  'qa',
+  'weather'
+);
+
+-- Time accounting charge discounts
+CREATE TABLE t_time_charge_discount (
+
+  c_id               bigserial   PRIMARY KEY,
+
+  c_visit_id         d_visit_id  NOT NULL REFERENCES t_visit(c_visit_id) ON DELETE CASCADE,
+
+  c_start            timestamp   NOT NULL,
+
+  c_end              timestamp   NOT NULL,
```

Comment:
Do these need to be constrained such that `c_start` <= `c_end`?

Reply:
Yes, thanks, I overlooked that one.